src/runtime/components/CookieControl.vue

```diff
@@ -257,17 +99,10 @@ const cookieCookiesEnabledIds = useCookie(
   },
 )
 
-// computations
-const isSaved = computed(
-  () =>
-    getCookieIds(cookiesEnabled.value || [])
-      .sort()
-      .join(COOKIE_ID_SEPARATOR) !==
-    getCookieIds(localCookiesEnabled.value).sort().join(COOKIE_ID_SEPARATOR),
-)
+// Computed
 const localeStrings = computed(() => moduleOptions.localeTexts[props.locale])
 
-// methods
+// Methods
```

Comment:
Let's extract composables for code that belongs together where it makes sense instead of changing the comments here. Those were only added for migration to Nuxt 3 some time back and they can be discarded once related code is extracted to composables. Background: https://www.youtube.com/watch?v=iKaDFAxzJyw

In this case the separate components you are creating are likely enough separation, no need for composables due to limited reuse of functionality, I'd say.

Reply:
Yeah I am currently making a way out how to structure the changes. I would prefer to implement the features from the issues and in a separat MR try to to some code-cleanup.

But I am completely with you I think we can definitely improve the code structure/format!